### PR TITLE
feature(RE-1763): Markdown in news

### PR DESCRIPTION
**Pull request by nitrique**

### CODE REVIEW

File: src/helpers/homeFeed.ts [hunk 1]
```diff
@@ -41,34 +42,42 @@ export const tranformFeedItemToProps = (feed: RestTimelineFeedItem): FeedCardPro
...
         type,
         onShare: () => {},
-        onShow: () => {
-          router.push({
-            pathname: '/(tabs)/(home)/news-detail',
-            params: { id: feed.objectID },
-          })
+        onShow: async () => {
+          if (feed.cta_link && (await Linking.canOpenURL(feed.cta_link))) {
+            await Linking.openURL(feed.cta_link)
```
Comment (OverGlass) [inline src/helpers/homeFeed.ts:60]: Peut etre ouvrvir avec expo webbrowser ?
https://docs.expo.dev/versions/latest/sdk/webbrowser/

Comment (nitrique) [reply src/helpers/homeFeed.ts]: 👍🏻

File: src/helpers/homeFeed.ts [hunk 2]
```diff
@@ -41,34 +43,42 @@ export const tranformFeedItemToProps = (feed: RestTimelineFeedItem): FeedCardPro
...
-          router.push({
-            pathname: '/(tabs)/(home)/news-detail',
-            params: { id: feed.objectID },
-          })
+        onShow: async () => {
+          if (feed.cta_link) {
+            await openBrowserAsync(feed.cta_link).catch(logDefaultError)
+          } else {
+            router.push({
+              pathname: '/(tabs)/(home)/news-detail',
```
Comment (OverGlass) [inline src/helpers/homeFeed.ts:64]: pathname: '/(tabs)/(home)/(modals)/news-detail',

Comment (nitrique) [reply src/helpers/homeFeed.ts]: done

**Review by OverGlass — APPROVED** [review]
LGTM

**Pull request merged**